static/app/components/dropdownMenuControlV2.tsx

```diff
@@ -121,6 +121,14 @@ function MenuControl({
     ref
   );
 
+  // Calculate the current trigger element's width. This will be used as
+  // the min width for the menu.
+  const [triggerWidth, setTriggerWidth] = useState<number>();
+  useEffect(() => {
```

Comment:
In addition to this, you may need to use `useResizeObserver()` in case the dropdown menu container is resized. 

Similar to this pattern: https://github.com/adobe/react-spectrum/blob/ebd6bc2c9269969181d978fd40fa84424edacf9e/packages/%40react-spectrum/buttongroup/src/ButtonGroup.tsx#L69-L80

Reply:
oh good point! Added in https://github.com/getsentry/sentry/pull/32061/commits/3f3ee40f9e1796da961c8d7fecb7417ca988661a